raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -192,7 +192,6 @@ public KafkaRaftClient(NetworkChannel channel,
         this.purgatory = purgatory;
         this.time = time;
         this.timer = time.timer(fetchTimeoutMs);    // initialize assuming it is an observer
-        this.retryBackoffMs = retryBackoffMs;
         this.electionTimeoutMs = electionTimeoutMs;
         this.electionJitterMs = electionJitterMs;
```

Comment:
Do we want to consider renaming `election.jitter.ms` or do you think the name is clear enough with the altered usage.

Reply:
yeah I'm thinking maybe rename to `election.max.backoff.ms`.